### PR TITLE
Explain the `--color` option in a comment

### PR DESCRIPTION
**Pull request by lydell**

### CODE REVIEW

File: bin/prettier.js [hunk 1]
```diff
@@ -14,7 +14,10 @@ const argv = minimist(process.argv.slice(2), {
     "single-quote",
     "trailing-comma",
     "bracket-spacing",
-    // See https://github.com/chalk/supports-color/#info
+    // The supports-color package (a sub sub dependency) looks directly at
+    // `process.argv` for `--no-color` and such-like options. The reason it is
+    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
+    // See https://github.com/chalk/supports-color/#info for more information.
```
Comment (lydell) [inline bin/prettier.js]: Hmm, actually, `color: true` is not strictly needed a few lines down, but after all – colors _are_ enabled by default. Should I remove it?

**Comment by vjeux** [comment]
sgtm. I'll let @jlongster merge it since he asked for it

File: bin/prettier.js (hunk 1 above)
Comment (jlongster) [inline bin/prettier.js]: I suppose we can keep it, but it is a little weird for a dependency for be reading straight from `process.argv`. I wish it didn't do that. But it's fine to document the default.

**Comment by jlongster** [comment]
Thanks!

**Pull request merged**